packages/components/src/components/languageSupport/squiggle.grammar

```diff
@@ -0,0 +1,104 @@
+
+@precedence {
```

Comment:
Shame this duplicates our existing peggy specifications. It probably has to be done though.

Reply:
IIRC @berekuk wants to remove peggy eventually, so tying it here to peggy probably wouldn't've been a good idea